projects/memapCore/src/main/java/memap/components/prototypes/Building.java

```diff
@@ -15,7 +15,6 @@
 import memap.helper.configurationOptions.ToolUsage;
```

Comment:
only cleanup, removed unused dependencies

Reply:
Here, I got an error during testing in live-mode and I had to comment out line 133 (this is why the dependency is not used anymore). I will take care of that in another branch.